docs/apache-airflow-providers-amazon/auth-manager/manage/index.rst

```diff
@@ -18,3 +18,205 @@
 ================================================
 Manage Airflow environment with AWS auth manager
 ================================================
+
+When the AWS auth manager is used, all users and their permissions are no longer managed by
+Airflow itself but by AWS through two different services: AWS IAM Identity Center (users) and Amazon Verified
```

Comment:
no longer managed by the Flask auth manager, which is default in Airflow, but by AWS-based authorization integration through...